languages/julia/config.json

```diff
@@ -8,6 +8,47 @@
         "indent_style": "space",
         "indent_size": 4
     },
+    "tags": [
+        "compiles_to/binary",
+        "typing/dynamic",
+        "paradigm/declarative",
+        "paradigm/functional",
+        "paradigm/imperative",
+        "paradigm/object_oriented",
+        "paradigm/procedural"
+    ],
+    "key_features": [
+        {
+            "title": "Fast",
+            "content": "Julia is designed for high performance. Code compiles to efficient native code for various platforms",
+            "icon": ""
```

Comment:
You could consider using a temporary icon (e.g. `features-oop`) or just add TODO?